patroni/api.py

```diff
@@ -252,8 +252,8 @@ def get_postgresql_status(self, retry=False):
     def get_tags(self):
         return {'tags': self.server.patroni.tags}
 
-    def log_message(self, format, *args):
-        logger.debug("API thread: " + format % args)
+    def log_message(self, fmt, *args):
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Method could be a function](https://www.codacy.com/app/cyberdemn/patroni/file/2033577187/issues/source?bid=3127594&fileBranchId=3129207#l255)